lib/WasmReader/Chakra.WasmReader.vcxproj

```diff
@@ -0,0 +1,71 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we have a Github issue / todo comment etc. that reminds us to implement this on cross-platform?